2.4/root/usr/share/container-scripts/mongodb/common.sh

```diff
@@ -86,8 +86,12 @@ function _wait_for_mongo() {
 # To get list of endpoints, you need to have headless Service named 'mongodb'.
 # NOTE: This won't work with standalone Docker container.
 function endpoints() {
-  service_name=${MONGODB_SERVICE_NAME:-mongodb}
-  dig ${service_name} A +search +short 2>/dev/null
+  if [ -n "${MONGODB_REPLICA_MEMBERS+x}" ]; then
```

Comment:
This is rather personal preference (the '+' vs '-'), so just nitpick.  In usual shell scripting the ':-' statement is used rather properly, here we have the ugly `nounset` so the ':-' is rather overused.